### PR TITLE
Update tech.html

### PR DESCRIPTION
**Pull request by boughtabook**
Added NLP to Tech section of website

### CODE REVIEW

File: tech.html [hunk 1]
```diff
@@ -81,6 +81,428 @@ <h1>Autonomous Vehicles (AV)</h1>
...
             <p>Having said all that, I wouldn’t use an autonomous car until they had been cleared as 100% reliable and safe. With so much computerisation involved, they are susceptible to hacking and malfunction. If I had the choice between a Level 5, fully automated with no access to a steering wheel and a Level 4, most functions automated except under unusual circumstances, I would choose Level 4, as I don’t completely trust the way the world is heading in relation to the advancements in technology. I feel like it is about to supersede the need for humans in every sector. Sometimes it’s important to stop and look around at what we’ve got before it’s gone forever.</p>
               <p>ADD REFERENCES</p>
+              <img src="img/NLP.png" alt="NLP" width="460" height="345">
+
+<p>
+
+    Natural Language processing and chatterbots
```
Comment (JDT-dev) [inline tech.html]: Will you be bolding this heading?

File: tech.html [hunk 2]
```diff
@@ -81,6 +81,428 @@ <h1>Autonomous Vehicles (AV)</h1>
...
+
+</p>
+
+<p>
+
+    <strong>What does it do?</strong>
```
Comment (JDT-dev) [inline tech.html]: I don't think we need to include the question headings. I didn't.

File: tech.html (hunk 1 above)
Comment (boughtabook) [reply tech.html]: Yes, we all need to format in the same way.

File: tech.html (hunk 2 above)
Comment (boughtabook) [reply tech.html]: I'll delete them.

**Comment by boughtabook** [comment]
Updated headings

**Pull request merged**